### ISSUE TITLE
Authentication required message

### ISSUE THREAD

**Issue by Gordonby** [a11ywatch/github-actions#13]
I am attempting to use this action on a site that has no authentication, but am receiving this error;

> Run a11ywatch scan --url https://azure.github.io/AKS-Construction/ --external --save
{"data":null,"message":"Authentication required. Add your authentication header and try again.","success":false}

## To reproduce

```yaml
name: Accessibility
on:
  workflow_dispatch:

jobs:
  accessibility:
    runs-on: ubuntu-latest
    steps:
      #- uses: actions/checkout@v3

      - uses: a11ywatch/github-action@v1.4.0
        name: Accessibility checks
        with:
          WEBSITE_URL: https://azure.github.io/AKS-Construction/
          FAIL_ERROR_COUNT: 10
          EXTERNAL: false
```

**Comment by j-mendez** [comment]
@Gordonby Hi, thank you for reporting the issue. `EXTERNAL` at the moment is handled in a way that any value assigned considers it true including a single white space " ". Remove the key from the config for now, we need to update it so it expects the key to be equal to true. The authentication method is for using the remote API. If the `EXTERNAL` config is set, this action will not startup the A11yWatch suite locally and will use the a11ywatch.com API to retrieve results. If the flag is not set the action will bring up the A11yWatch service in a docker container to use locally.

**Comment by j-mendez** [comment]
Fixed in v1.4.1, thank you @Gordonby for the issue! https://github.com/A11yWatch/github-action/pull/14
